app/pages/BeelieversAuction/BeelieversBid.tsx

```diff
@@ -242,9 +241,9 @@ export function BeelieversBid({ user, entryBidMist }: BeelieversBidProps) {
 
 function submitButton(isPending: boolean, hasUserBidBefore: boolean) {
 	return (
-		<Button
+		<button
 			disabled={isPending}
-			className="h-14 lg:h-16 text-lg font-semibold bg-gradient-to-r from-primary to-orange-400 hover:from-orange-400 hover:to-primary transition-all duration-300 transform hover:scale-[1.02] shadow-lg hover:shadow-xl disabled:opacity-50 disabled:cursor-not-allowed"
+			className="btn btn-primary h-14 lg:h-16 text-lg font-semibold bg-gradient-to-r from-primary to-orange-400 hover:from-orange-400 hover:to-primary transition-all duration-300 transform hover:scale-[1.02] shadow-lg hover:shadow-xl disabled:opacity-50 disabled:cursor-not-allowed"
```

Comment:
can we have a function to add this effects? The function will return css classes.

Reply:
let's call them effect classes

Reply:
Done